src/plugins/azgenerator/templates/CliReport.ts

```diff
@@ -1,168 +1,300 @@
-﻿/*---------------------------------------------------------------------------------------------
+/*---------------------------------------------------------------------------------------------
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { SchemaType } from "@azure-tools/codemodel";
-import { isNullOrUndefined } from "util";
 import { CodeModelAz } from "../CodeModelAz";
+import { ParameterLocation, SchemaType } from "@azure-tools/codemodel";
+import { EscapeString,  ToCamelCase, Capitalize,  ToMultiLine, ToJsonString } from "../../../utils/helper"
+import { isNullOrUndefined } from "util";
```

Comment:
Is it possible to support generating report in incremental mode?

Reply:
The hidden command and commandgroup will be filtered by the Select method. the report generator doesn't need to change.